pkg/kubelet/pluginmanager/BUILD

```diff
@@ -0,0 +1,56 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_library", "go_test")
```

Comment:
Oh and before I forget, since this is a significant new component, add an OWNERS file at `pkg/kubelet/pluginmanager/OWNERS` and add yourself and me (`saad-ali`) as approvers. Example: https://github.com/kubernetes/kubernetes/blob/master/pkg/controller/volume/attachdetach/OWNERS